src/components/views/rooms/BasicMessageComposer.tsx

```diff
@@ -105,6 +106,7 @@ interface IState {
     completionIndex?: number;
 }
 
+@replaceableComponent("views.rooms.BasixMessageEditor")
```

Comment:
Wow, I caught one! 😜 

```suggestion
@replaceableComponent("views.rooms.BasicMessageEditor")
```